starter_scripts/build_docker_runner_image.sh

```diff
@@ -2,11 +2,19 @@
 
 set -euo pipefail
 
-SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+[[ "$(uname)" = Darwin ]] && rl=greadlink || rl=readlink
```

Comment:
Not sure, if I like this construct. It is shorter than an if, but I feel it has more potential for errors.